lib/jekyll-feed/generator.rb

```diff
@@ -44,12 +51,14 @@ def file_exists?(file_path)
     end
 
     # Generates contents for a file
-    def content_for_file(file_path, file_source_path)
+
+    def content_for_file(file_path, file_source_path, category = false)
       file = PageWithoutAFile.new(@site, __dir__, "", file_path)
       file.content = File.read(file_source_path).gsub(MINIFY_REGEX, "")
       file.data["layout"] = nil
       file.data["sitemap"] = false
       file.data["xsl"] = file_exists?("feed.xslt.xml")
+      file.data["category"] = category
```

Comment:
Would it be cleaner to allow the template to accept an arbitrary array of documents? That way, we can pass it all post, a category, a tag, or even another collection.